config/__init__.py

```diff
@@ -4,12 +4,67 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-'''This module handles application configuration and secrets.'''
+"""This module handles application configuration and secrets."""
 
 from collections import namedtuple
+import json
+import os
+import sys
+import urlparse
+
+import requests
+
+from environment import *  # nopep8
+from secrets import *  # nopep8
+
+
+def initialize_status():
+    """Map the status name with the milestone id on GitHub.
+
+    The project needs the mapping in between milestones and their id
+    to be able to query status related things. It uses a backup version
+    when the request to GitHub fails.
+    """
+    print('Statuses Initialization…')
+    REPO_ROOT = ISSUES_REPO_URI.rpartition('/issues')[0]
+    milestones_url_path = os.path.join('repos', REPO_ROOT, 'milestones')
+    milestones_url = urlparse.urlunparse(
+        ('https', 'api.github.com', milestones_url_path, '', '', ''))
+    milestones_path = os.path.join(DATA_PATH, 'milestones.json')
+    try:
+        # Get the milestone from the network
+        print('Fetching milestones from Github…')
+        r = requests.get(milestones_url)
+        milestones_content = r.content
+        if r.status_code == 200:
+            with open(milestones_path, 'w') as f:
+                f.write(r.content)
+            print('Milestones saved in data/')
+        r.raise_for_status()
+    except requests.exceptions.HTTPError as error:
+        # Not working, let's use the cached copy
+        # This might fail the first time.
+        print(MILESTONE_ERROR.format(msg=error))
```

Comment:
Ah, this will probably throw. `MILESTONE_ERROR` is defined in run.py. We should fix this before merging.

Reply:
OK done.